drivers/fpga/Kconfig.eos_s3

```diff
@@ -0,0 +1,9 @@
+# FPGA EOS_S3 driver configuration options
+
+# Copyright (c) 2021 Antmicro <www.antmicro.com>
+# SPDX-License-Identifier: Apache-2.0
+
+config EOS_S3_FPGA
+	bool "EOS_S3 fpga driver"
+	help
+	  Enable EOS_S3 FPGA driver.
```

Comment:
Could we have the FPGA name without the underscore? I.e.:
> Enable EOS S3 FPGA driver.
As this seems to be the name that the vendor uses: https://www.quicklogic.com/products/soc/

Reply:
Good point, I changed the name to without underscore.